docs/test-fairy/sdk/tf-production.md

```diff
@@ -8,11 +8,162 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+### iOS 
+
+#### Option 1: Calling [TestFairy begin] only in DEBUG
+
+Without a call to [TestFairy begin], the SDK is not initialized. An uninitialized SDK won't consume any memory, won't open sockets, and won't catch uncaught exceptions. Even though it does not impact your app in any way, the SDK is still linked with your app. This is the easiest option.
+
+##### Objective-C
+```
+#ifdef DEBUG
+[TestFairy begin:@"APP_TOKEN"];
+#endif
+```
+
+##### Swift
+```
+#if DEBUG
+TestFairy.begin("APP_TOKEN")
+#endif
+```
+
+If your publishing workflow has multiple build schemes or you plan to implement such phases, please proceed to [this post](https://blog.testfairy.com/ios-build-schemes-explained/) to learn how to do that.
+
+We suggest defining a compiler flag for each scheme you have to enable the SDK for schemes relevant to testing like below.
+
+##### Objective-C
+```
+#if defined(DEBUG)
+[TestFairy begin:@"APP_TOKEN"];
+#elif defined(SCHEME1)
+[TestFairy begin:@"APP_TOKEN"];
+#elif defined(SCHEME2)
+[TestFairy begin:@"APP_TOKEN"];
+#else
+// Don't initialize the SDK
+#endif
+```
+
+##### Swift
+```
+#if DEBUG
+TestFairy.begin("APP_TOKEN")
+#elseif SCHEME1
+TestFairy.begin("APP_TOKEN")
+#elseif SCHEME2
+TestFairy.begin("APP_TOKEN")
+#else
+// Don't initialize the SDK
+#endif
+```
+
+If you are also worried about reducing the app size in your final release build, proceed to Option 2.
+
+#### Option 2: Configure link options in a scheme for App Store
+
+A common pattern we see from our customers is having a dedicated scheme for App Store. Meaning there's a Debug, Release and App Store (and maybe others.)
+
+The link phases that are explained in the integration document only apply to Debug and Release schemes, and can be excluded from the App Store scheme.
+
+This solution still requires use of `#ifdef` or `#if` as before, but can also completely omit the library from being linked with the app.
+
+Navigate to project build settings and locate **Excluded Source File Name** option. Expand the list and find the build scheme you want to exclude TestFairy from. Double click the row add two entries to the excluded file list, one for **TestFairy.h**, one for **libTestFairy.a** files.
+
+Try building your project. If the compilation fails, locate the lines where TestFairy is used and wrap them with `#ifdef` or `#if` directives explained in Option 1.
+
+<a name="ios-noop"></a>
+### Option 3: No-op SDK
+
+Similar to Option #2, you're required to have multiple schemes in your project, but **does not** require the use of `#ifdef` or `#if`.
+
+1. Download the [TestFairy iOS No-Op SDK](https://github.com/testfairy/testfairy-ios-sdk-noop)
+2. Add `TestFairy.m` to your project's *Production* or *App Store* scheme. (Note, that the `.m` file needs to be put in a place where it can find the `TestFairy.h` file).
```

Comment:
```suggestion
2. Add `TestFairy.m` to your project's *Production* or *App Store* scheme. (Note that the `.m` file needs to be placed where it can find the `TestFairy.h` file).
```